test/test_transformations.jl

```diff
@@ -24,20 +24,6 @@ psi_n = coherentstate(b_fock, α0)
 psi_x = gaussianstate(b_position, x0, p0, 1)
 @test 1e-10 > D(psi_x, Txn*psi_n)
 
-# Test different characteristic length
-x0 = 0.0
-p0 = 0.2
-α0 = (x0 + 1im*p0)/sqrt(2)
-σ0 = 0.7
-Txn = transform(b_position, b_fock; x0=σ0)
-Tnx = transform(b_fock, b_position; x0=σ0)
-@test 1e-10 > D(dagger(Txn), Tnx)
-@test 1e-10 > D(one(b_fock), Tnx*Txn)
-
-psi_n = coherentstate(b_fock, α0)
-psi_x = gaussianstate(b_position, x0/σ0, p0/σ0, σ0)
-@test 1e-10 > D(psi_x, Txn*psi_n)
```

Comment:
Could you elaborate on why these are deleted?

Reply:
Ah yes I forgot to mention in the PR that I also changed the interface for `transform`. It would be easy to add this back in but given this optional x0 argument wasn't documented and it wasn't at all clear how it would be used, it was easier for me to remove it as I was working on this.